gpflow/conditionals/util.py

```diff
@@ -441,8 +441,9 @@ def fully_correlated_conditional_repeat(
         if q_sqrt.shape.ndims == 3:
             A_tiled = tf.tile(A[None, :, :], tf.stack([R, 1, 1]))  # [R, M, P]
             LTA = tf.linalg.matmul(Lf, A_tiled, transpose_a=True)  # [R, M, P]
-        elif q_sqrt.shape.ndims == 2:  # pragma: no cover
-            raise NotImplementedError("Does not support diagonal q_sqrt yet...")
+        elif q_sqrt.shape.ndims == 2:
+            A_tiled = tf.tile(A[None, :, :], tf.stack([R, 1, 1]))  # [R, M, P]
+            LTA = Lf * A_tiled  # [R, M, P]
```

Comment:
Should this be `Lm` instead of `Lf`? Seems like this path might not be getting tested...

Reply:
nevermind, I thought there was no Lf defined but I missed it, it is in line 443. :)